xbmc/windowing/osx/OpenGL/OSXGLWindow.mm

```diff
@@ -64,19 +64,18 @@ - (void)windowDidExpose:(NSNotification*)aNotification
 
 - (void)windowDidMove:(NSNotification*)aNotification
 {
-  NSOpenGLContext* context = NSOpenGLContext.currentContext;
-  if (context)
+  NSWindow* window = [aNotification object];
+  if (window.contentView)
   {
-    if (context.view)
+    NSPoint window_origin = [window.contentView frame].origin;
```

Comment:
if `window.contentView` is nil, then the value returned by `frame` is undefined (nil receivers cannot return structs).

You have to check for a non-nil receiver before accessing selectors that return structs such as `NSRect`.

Reply:
there's `if (window.contentView)` just above :)

Reply:
Yes, so it is.  I got distracted by the `[[[context view] window] frame] ` being replaced.